src/index.js

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import { Provider } from 'react-redux';
-import BookStore from 'components/Books';
+import Books from './Components/BookStore';
 import store from './redux/configureStore';
 
 ReactDOM.render(
   <React.StrictMode>
     <Provider store={store}>
-      <BookStore />
+      <Books />
     </Provider>
   </React.StrictMode>,
   document.getElementById('root'),
```

Comment:
- [ ] Although I am highlighting this piece of code, the changes I am about to request are not connected to them. According to the requirement, the `Categories` component should render a `button`  with the text `"Check Status"`. Kindly make this change. 👍